hazelcast/src/main/java/com/hazelcast/internal/partition/impl/PartitionPrimaryReplicaAntiEntropyTask.java

```diff
@@ -46,6 +49,10 @@ public void run() {
                 invokePartitionBackupReplicaAntiEntropyOp(index, replica, namespaces, null);
             }
         }
+
+        if (afterRun != null) {
+            afterRun.run();
```

Comment:
Shouldn't `afterRun.run()` be called inside a `finally` block? What if this method returns before executing `afterRun`?

Reply:
fixed